dev-qt/qtgui/qtgui-5.8.9999.ebuild

```diff
@@ -137,12 +138,14 @@ src_prepare() {
 	use dbus || sed -i -e 's/contains(QT_CONFIG, dbus)/false/' \
 		src/platformsupport/platformsupport.pro || die
 
+	qt_use_disable_config tuio udpsocket src/plugins/generic/generic.pro
+
 	qt_use_disable_mod ibus dbus \
```

Comment:
(minor thing) the "avoid automagic dep on qtdbus" comment above was supposed to refer to this line too... while the "avoid automagic dep on qtnetwork" still applies to the `tuio` stuff that you moved up...